packages/eslint-plugin/src/rules/no-styled-tagged-template-expression/__tests__/rule.test.ts

```diff
@@ -899,5 +899,84 @@ tester.run('no-styled-tagged-template-expression', noStyledTaggedTemplateExpress
         );
       `,
     },
+    {
+      filename: 'multiple-selectors-across-lines.ts',
+      code: `
+        import { styled } from '@compiled/react';
+
+        styled.div\`
+          color: blue;
+          &:hover,
+          &:focus,
+          &:active {
+            text-decoration: inherit;
+          }
+          ul,
+          li {
+            color: red;
+          }
+        \`;
+      `,
+      output: `
+        import { styled } from '@compiled/react';
+
+        styled.div({
+          color: "blue",
+          "&:hover, &:focus, &:active": {
+            textDecoration: "inherit"
+          },
+          "ul, li": {
+            color: "red"
+          }
+        });
+      `,
+    },
+    {
+      filename: 'multiple-selectors-on-same-line.ts',
+      code: `
+        import { styled } from '@compiled/react';
+
+        styled.div\`
+          color: blue;
+          &:hover, &:focus,
+          &:active {
+            text-decoration: inherit;
+          }
+          ul, li {
+            color: red;
+          }
+        \`;
+      `,
+      output: `
+        import { styled } from '@compiled/react';
+
+        styled.div({
+          color: "blue",
+          "&:hover, &:focus, &:active": {
+            textDecoration: "inherit"
+          },
+          "ul, li": {
+            color: "red"
+          }
+        });
+      `,
+    },
+    {
+      filename: 'do-not-handle-invalid-css.ts',
+      code: `
+        import { styled } from '@compiled/react';
+
+        styled.div\`
+          color blue;
+        \`;
+      `,
+      output: `
+        import { styled } from '@compiled/react';
+
+        styled.div\`
+          color blue;
+        \`;
+      `,
+    },
   ]),
```

Comment:
Can you please also add a test for https://github.com/atlassian-labs/compiled/pull/1313#discussion_r1027385229